packages/api/cms-api/src/dependencies/dto/dependencies.args.ts

```diff
@@ -1,24 +1,33 @@
 import { ArgsType, Field } from "@nestjs/graphql";
 import { Type } from "class-transformer";
-import { IsOptional, ValidateNested } from "class-validator";
+import { ValidateNested } from "class-validator";
 
 import { OffsetBasedPaginationArgs } from "../../common/pagination/offset-based.args";
+import { IsUndefinable } from "../../common/validators/is-undefinable";
 import { DependencyFilter, DependentFilter } from "./dependencies.filter";
 
 @ArgsType()
 export class DependenciesArgs extends OffsetBasedPaginationArgs {
     @Field(() => DependencyFilter, { nullable: true })
     @ValidateNested()
     @Type(() => DependencyFilter)
-    @IsOptional()
+    @IsUndefinable()
     filter?: DependencyFilter;
+
+    @Field(() => Boolean, { nullable: true, defaultValue: false })
+    @IsUndefinable()
+    force: boolean;
 }
 
 @ArgsType()
 export class DependentsArgs extends OffsetBasedPaginationArgs {
     @Field(() => DependentFilter, { nullable: true })
     @ValidateNested()
     @Type(() => DependentFilter)
-    @IsOptional()
+    @IsUndefinable()
     filter?: DependentFilter;
+
+    @Field(() => Boolean, { nullable: true, defaultValue: false })
+    @IsUndefinable()
+    force: boolean;
```

Comment:
force in the graphql api is a bit strange parameter name, alternatives:

- forceRefresh
- refresh
- update
- forceUpdate